userguide/virtualmachines.rst

```diff
@@ -61,7 +61,68 @@ and |ui-add| to open the wizard shown in
 .. figure:: images/virtual-machines-add-wizard-type.png
 
    Add VM
-
+   
+   
+Create a virtual machine by following these seven steps of the creation
+wizard:
+
+#. **Select VM wizard type**
+
+   Select :literal:`Virtual Macine (VM)` or
+   :literal:`Docker Host`.
+
+#. **Operating System**
+
+   Select the guest operating system type from the drop down menu and
+   enter an alphanumeric name for the VM. It is recommended to use UEFI
+   as the boot method for newer operating systems. The settings
+   :guilabel:`Start on Boot` and :guilabel:`Enable VNC` are set by default.
+   The setting :guilabel:`Start on Boot` tells the VM to start when the
+   host system is booted. The setting :guilabel:`Enable VNC` allows a
+   virtual connection to the VM.
+   
+#. **CPU and Memory**
+
+   Enter the amount of virtual CPUs for the VM to use. Enter the amount
+   of memory, in MiB,for the VM to use. Not using adequate memory or
+   CPUs can affect the performance of the vm. Using too much memory or
+   CUPs can affect the performance of the host system.
+   
+#. **Hard Disks**
+
+   Select :guilabel:`Create new disk image` to create a new zvol on an
+   existing dataset. This is used as a virtual hard drive for the VM. Select
```

Comment:
Does the "on an existing dataset" really help?  I think this can be merged with the next sentence:
`` Select :guilabel:`Create new disk image` to create a new zvol to use as a virtual hard drive for the VM. ``